### ISSUE TITLE
Travis still attempting to build some map repos

### ISSUE THREAD

**Issue by ssoloff** [triplea-maps/Project#89]
I was under the impression we disabled Travis for all map repos a few months back.  Therefore, I was a bit surprised when I received a notification indicating the Travis build for triplea-maps/new_world_order-skin_gudkarma_1 [errored](https://travis-ci.org/triplea-maps/new_world_order-skin_gudkarma_1/builds/291380151).

According to the triplea-maps organization settings on Travis, it looks like the repos are disabled, but there also appear to be other repos that are enabled whose name only differs by case.  Not sure if that's an issue.

We should probably clean up the Travis settings and ensure all repos are disabled.  In addition, we should consider removing _.travis.yml_ and _build.gradle_ from all map repos.

@DanVanAtta @ron-murhammer

**Comment by ssoloff** [comment]
There appears to be a [CircleCI configuration](https://circleci.com/gh/triplea-maps/total_ancient_war/8?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link) for total_ancient_war.  That will need to be disabled, as well.

**Comment by ssoloff** [comment]
@DanVanAtta @ron-murhammer I have a script ready to go that will delete any _.travis.yml_ and _build.gradle_ file in all map repos.  **Are there any objections to removing those files?**

For reference, the script I plan on running follows:

```sh
#!/bin/bash -e

BASE_DIR=$(pwd)
#RM_DRY_RUN=--dry-run
#COMMIT_DRY_RUN=--dry-run

find . -name '.travis.yml' | while read line; do
  FOLDER=$(echo $line | sed s/\.travis\.yml//)
  echo "Removing '.travis.yml' from $FOLDER"
  cd $FOLDER
  git rm $RM_DRY_RUN .travis.yml
  cd $BASE_DIR
done

find . -name 'build.gradle' | while read line; do
  FOLDER=$(echo $line | sed s/build\.gradle//)
  echo "Removing 'build.gradle' from $FOLDER"
  cd $FOLDER
  git rm $RM_DRY_RUN build.gradle
  cd $BASE_DIR
done

find . -mindepth 1 -maxdepth 1 -type d | while read line; do
  FOLDER=$line
  cd $FOLDER
  if [[ $(git status --porcelain) ]]; then
    echo "Committing changes in $FOLDER"
    git commit $COMMIT_DRY_RUN -a -m "Remove build scripts"
  fi
  cd $BASE_DIR
done
```

A separate script will be run to execute `git push` in each repo.  I plan on running the scripts directly on `master`.

The changes that will be made by running the above script are summarized below:

```
Git: Status for ./1914-cow-empires
D  .travis.yml
D  build.gradle
Git: Status for ./sengoku_jidai
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order
D  .travis.yml
D  build.gradle
Git: Status for ./the_great_northern_war
D  .travis.yml
D  build.gradle
Git: Status for ./ultimate_world
D  .travis.yml
D  build.gradle
Git: Status for ./diplomacy-map_skin3
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order1915lebowski
D  .travis.yml
D  build.gradle
Git: Status for ./the_rising_sun
D  .travis.yml
D  build.gradle
Git: Status for ./world_at_war-skin_sneak
D  .travis.yml
D  build.gradle
Git: Status for ./cold_war_asia1948
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_v3
D  .travis.yml
D  build.gradle
Git: Status for ./ultimate_world_variants
D  .travis.yml
D  build.gradle
Git: Status for ./ww2_phillipines
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order-skin_sieg_2
D  .travis.yml
D  build.gradle
Git: Status for ./world_war2010
D  .travis.yml
D  build.gradle
Git: Status for ./europe
D  .travis.yml
D  build.gradle
Git: Status for ./sleeping_giant
D  .travis.yml
D  build.gradle
Git: Status for ./classic_variations
D  .travis.yml
D  build.gradle
Git: Status for ./tiberian_conquest_sa
D  .travis.yml
D  build.gradle
Git: Status for ./capture_the_flag
D  .travis.yml
D  build.gradle
Git: Status for ./big_world_variations
D  .travis.yml
D  build.gradle
Git: Status for ./war_of_the_lance
D  .travis.yml
D  build.gradle
Git: Status for ./roll_through_the_reich
Git: Status for ./conquest_of_the_world
D  .travis.yml
D  build.gradle
Git: Status for ./dragon_war
Git: Status for ./world_war_ii_revised
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order-skin_sneak
D  .travis.yml
D  build.gradle
Git: Status for ./age_of_tribes
Git: Status for ./feudal_japan
D  .travis.yml
D  build.gradle
Git: Status for ./roman_invasion
D  .travis.yml
D  build.gradle
Git: Status for ./eastern_front
D  .travis.yml
D  build.gradle
Git: Status for ./game_of_thrones
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order-skin_pulicat_1
D  .travis.yml
D  build.gradle
Git: Status for ./battle_of_aventurica
D  .travis.yml
D  build.gradle
Git: Status for ./iron_war
Git: Status for ./ww2v3_11n
D  .travis.yml
D  build.gradle
Git: Status for ./nwo_variants
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_revised_variations
D  .travis.yml
D  build.gradle
Git: Status for ./total_world_war
D  .travis.yml
D  build.gradle
Git: Status for ./d-day
D  .travis.yml
D  build.gradle
Git: Status for ./operation_market_garden
D  .travis.yml
D  build.gradle
Git: Status for ./napoleonic_empires-topographical_map_skin
D  .travis.yml
D  build.gradle
Git: Status for ./rome_total_war
D  .travis.yml
D  build.gradle
Git: Status for ./middle_earth-map_skin2
D  .travis.yml
D  build.gradle
Git: Status for ./civil_war-bigmap
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_europe
D  .travis.yml
D  build.gradle
Git: Status for ./domination_1914_blood_and_steel
D  .travis.yml
D  build.gradle
Git: Status for ./steampunk
D  .travis.yml
D  build.gradle
Git: Status for ./global_war
D  .travis.yml
D  build.gradle
Git: Status for ./diplomacy-map_skin1
D  .travis.yml
D  build.gradle
Git: Status for ./minimap
D  .travis.yml
D  build.gradle
Git: Status for ./1941
D  .travis.yml
D  build.gradle
Git: Status for ./ww2v3_variants
D  .travis.yml
D  build.gradle
Git: Status for ./cold_war
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order-skin_gudkarma_1
D  .travis.yml
D  build.gradle
Git: Status for ./camp_david
D  .travis.yml
D  build.gradle
Git: Status for ./diplomacy-map_skin4
D  .travis.yml
D  build.gradle
Git: Status for ./caribbean_trade_war
Git: Status for ./world_war_ii_v6_1941
D  .travis.yml
D  build.gradle
Git: Status for ./iron_war_europe
Git: Status for ./greyhawk
D  .travis.yml
D  build.gradle
Git: Status for ./star_wars_galactic_war
Git: Status for ./jurassic
D  .travis.yml
D  build.gradle
Git: Status for ./270bc
D  .travis.yml
D  build.gradle
Git: Status for ./first_punic_war
D  .travis.yml
D  build.gradle
Git: Status for ./arnhem
D  .travis.yml
D  build.gradle
Git: Status for ./total_ancient_war
D  .travis.yml
D  build.gradle
Git: Status for ./domination_1914_no_mans_land
D  .travis.yml
D  build.gradle
Git: Status for ./command_and_conquer
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_global
D  .travis.yml
D  build.gradle
Git: Status for ./elemental_forces
D  .travis.yml
D  build.gradle
Git: Status for ./big_world
D  .travis.yml
D  build.gradle
Git: Status for ./invasion_usa
D  .travis.yml
D  build.gradle
Git: Status for ./war_of_the_relics
D  .travis.yml
D  build.gradle
Git: Status for ./pact_of_steel_variations
D  .travis.yml
D  build.gradle
D  map/build.gradle
Git: Status for ./world_war_ii_pacific
D  .travis.yml
D  build.gradle
Git: Status for ./270bc-old_skin
D  .travis.yml
D  build.gradle
Git: Status for ./world_at_war_variants
D  .travis.yml
D  build.gradle
Git: Status for ./feudal_japan_warlords
D  .travis.yml
D  build.gradle
Git: Status for ./blue_vs_gray
D  .travis.yml
D  build.gradle
Git: Status for ./big_world_2
D  .travis.yml
D  build.gradle
Git: Status for ./pacific_challenge
D  .travis.yml
D  build.gradle
Git: Status for ./caravan
D  .travis.yml
D  build.gradle
Git: Status for ./greyhawk_wars
D  .travis.yml
D  build.gradle
Git: Status for ./Project
Git: Status for ./d-day2
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order_lebowski_edition
D  .travis.yml
D  build.gradle
Git: Status for ./global_war2
D  .travis.yml
D  build.gradle
Git: Status for ./world_at_war
D  .travis.yml
D  build.gradle
Git: Status for ./diplomacy-map_skin2
D  .travis.yml
D  build.gradle
Git: Status for ./red_sun_over_china
D  .travis.yml
D  build.gradle
Git: Status for ./peninsula_campaign
D  .travis.yml
D  build.gradle
Git: Status for ./ancient_times
D  .travis.yml
D  build.gradle
Git: Status for ./cold_war-1965
D  .travis.yml
D  build.gradle
Git: Status for ./cybertank
D  .travis.yml
D  build.gradle
Git: Status for ./civil_war
D  .travis.yml
D  build.gradle
Git: Status for ./270bc_variants
Git: Status for ./new_world_order_variants
D  .travis.yml
D  build.gradle
Git: Status for ./middle_earth
D  .travis.yml
D  build.gradle
Git: Status for ./the_pact_of_steel
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_classic
D  .travis.yml
D  build.gradle
Git: Status for ./tiberian_conquest_eu
D  .travis.yml
D  build.gradle
Git: Status for ./new_world_order-upsidedown
D  .travis.yml
D  build.gradle
Git: Status for ./star_wars_tatooine_war
Git: Status for ./tactics_campaign
D  .travis.yml
D  build.gradle
Git: Status for ./global_1940_redesign_house_rules
D  .travis.yml
D  build.gradle
Git: Status for ./small_balanced_2_or_4_player
D  .travis.yml
D  build.gradle
Git: Status for ./great_war
D  .travis.yml
D  build.gradle
Git: Status for ./twilight_imperium
D  .travis.yml
D  build.gradle
Git: Status for ./star_trek_dilithium_war
Git: Status for ./middle_east
D  .travis.yml
D  build.gradle
Git: Status for ./aa_enhanced_revised
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_v5_1942
D  .travis.yml
D  build.gradle
Git: Status for ./stalingrad
D  .travis.yml
D  build.gradle
Git: Status for ./neuschwabenland
D  .travis.yml
D  build.gradle
Git: Status for ./age_of_the_sturlungs
D  .travis.yml
D  build.gradle
Git: Status for ./tiberian_conquest_au
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_v4
D  .travis.yml
D  build.gradle
Git: Status for ./empire
D  .travis.yml
D  build.gradle
Git: Status for ./diplomacy
D  .travis.yml
D  build.gradle
Git: Status for ./napoleonic_empires
D  .travis.yml
D  build.gradle
Git: Status for ./great_lakes_war
D  .travis.yml
D  build.gradle
Git: Status for ./pacific
D  .travis.yml
D  build.gradle
Git: Status for ./battle_of_jutland
D  .travis.yml
D  build.gradle
Git: Status for ./pacific_1942
D  .travis.yml
D  build.gradle
Git: Status for ./domination
D  .travis.yml
D  build.gradle
Git: Status for ./domination_1914-weltpolitik
Git: Status for ./ur_quan_war_masters_edition
D  .travis.yml
D  build.gradle
Git: Status for ./zombieland
D  .travis.yml
D  build.gradle
Git: Status for ./hex_globe10
D  .travis.yml
D  build.gradle
Git: Status for ./map_making_tutorial
D  .travis.yml
D  build.gradle
Git: Status for ./tutorial
D  .travis.yml
D  build.gradle
Git: Status for ./blood_and_iron
D  .travis.yml
D  build.gradle
Git: Status for ./stellar_forces
D  .travis.yml
D  build.gradle
Git: Status for ./atari
D  .travis.yml
D  build.gradle
Git: Status for ./world_war_ii_global-battlemap_skin
D  .travis.yml
D  build.gradle
Git: Status for ./large_middle_earth
D  .travis.yml
D  build.gradle
```

**Comment by ssoloff** [comment]
@DanVanAtta @ron-murhammer Bumping...

**Comment by ssoloff** [comment]
@DanVanAtta @ron-murhammer Bumping...

**Comment by ssoloff** [comment]
@DanVanAtta @ron-murhammer Bumping...

**Comment by ron-murhammer** [comment]
@ssoloff No, objections from me as that is my understanding as well. But @DanVanAtta set that up so would probably know for sure.

**Comment by ssoloff** [comment]
Thanks, @ron-murhammer.  I'll wait for @DanVanAtta then.

**Comment by ssoloff** [comment]
@DanVanAtta Bumping...

**Comment by ssoloff** [comment]
@DanVanAtta Bumping...

**Comment by DanVanAtta** [comment]
Those are good cleanups, go for them when you can.

For map repo's, I generally have advised makers of admin changes to give everyone a chance to push changes first so they do not have to do any kind of merging. I suspect you are good to go ahead, though that is generally the one 'hidden' consideration I would have that I can think of.

**Comment by ssoloff** [comment]
@DanVanAtta Good point.  In this case, I believe all these changes will touch files that map authors would never modify, so the chance for a merge conflict should be relatively close to nil.

**Comment by DanVanAtta** [comment]
Agree, though the bar is even lower, it's the act of even needing to update because the push is blocked due to current branch not being latest. We lost some people even at "create a github account", so the git difficulty bar is as low as possible.

In practice, looking at active repo's: https://github.com/triplea-maps, nothing terribly active at the moment, a message on gitter for map admins to update would be useful perhaps. Forums is more appropriate even, but seems like overkill for this kind of FYI.

IMO go ahead with the updates.

**Comment by ssoloff** [comment]
Ah, I see.  I took it for granted a map author would have some minimal Git knowledge to sync their local repo with the remote.

Agree a notification is probably overkill for this change, but I'll probably do it anyway simply to record the fact that it happened in someplace public since I'll be committing directly to `master` without any PRs.

**Comment by ssoloff** [comment]
@ron-murhammer @DanVanAtta

In order to complete this task, I need push access to the following map repos:

* https://github.com/triplea-maps/270bc_variants
* https://github.com/triplea-maps/domination_1914-weltpolitik
* https://github.com/triplea-maps/iron_war
* https://github.com/triplea-maps/iron_war_europe
* https://github.com/triplea-maps/new_world_order-skin_sneak

TIA!

**Comment by ssoloff** [comment]
@Cernelius @FrostionAAA @sneakingcoward I believe each of you may have admin rights to one or more of the above repos.  If so, could you please grant me push access to the repo(s) you control?

**Comment by ron-murhammer** [comment]
@ssoloff You should have write access now. I needed to add those new repos to the admin team.

**Comment by ssoloff** [comment]
@ron-murhammer Confirmed.  Thanks!

**Comment by sneakingcoward** [comment]
get in contact with me directly please.

 

sneakingcoward <wgbt@gmx.at>

 

 

 

Gesendet: Mittwoch, 13. Dezember 2017 um 05:43 Uhr
Von: "Steven Soloff" <notifications@github.com>
An: triplea-maps/Project <Project@noreply.github.com>
Cc: sneakingcoward <wgbt@gmx.at>, Mention <mention@noreply.github.com>
Betreff: Re: [triplea-maps/Project] Travis still attempting to build some map repos (#89)


@Cernelius @FrostionAAA @sneakingcoward I believe each of you may have admin rights to one or more of the above repos. If so, could you please grant me push access to the repo(s) you control?

**Comment by sneakingcoward** [comment]
please for

 

waw variants

nwo variants

 

no write or change access for other persons then admins please...

 

thank you.

 

 

Gesendet: Mittwoch, 13. Dezember 2017 um 05:48 Uhr
Von: Ron <notifications@github.com>
An: triplea-maps/Project <Project@noreply.github.com>
Cc: sneakingcoward <wgbt@gmx.at>, Mention <mention@noreply.github.com>
Betreff: Re: [triplea-maps/Project] Travis still attempting to build some map repos (#89)


@ssoloff You should have write access now. I needed to add those new repos to the admin team.

**Comment by FrostionAAA** [comment]
So I guess it is all OK now? No need for us to do any file removing? Thanks for your maintenance work here at github 😃👍

**Comment by sneakingcoward** [comment]
for

waw variants

nwo variants

 

please no other access rights.

 

will do change requests if possible all myself.

it makes no sense, when more persons are changing one map without coordination.

one map creator per map should be enough...

 

thank you.

 

Gesendet: Mittwoch, 13. Dezember 2017 um 14:56 Uhr
Von: FrostionAAA <notifications@github.com>
An: triplea-maps/Project <Project@noreply.github.com>
Cc: sneakingcoward <wgbt@gmx.at>, Mention <mention@noreply.github.com>
Betreff: Re: [triplea-maps/Project] Travis still attempting to build some map repos (#89)


So I guess it is all OK now? No need for us to do any file removing? Thanks for your maintenance work here at github 😃👍

**Comment by ssoloff** [comment]
@FrostionAAA Yes, I now have push access to all repos, so there's nothing that you need to do.  Sorry for the poke. :smile:

@sneakingcoward The changes I will be submitting will not touch any map resources.  They will only delete the _.travis.yml_ and _build.gradle_ files.  If you'd prefer to do that yourself, please let me know which repos you manage, and I'll exclude them from my script.  Otherwise, I plan on doing a mass change to all repos sometime today.

**Comment by ssoloff** [comment]
The _.travis.yml_ and _build.gradle_ files have been removed from all map repos.

@ron-murhammer @DanVanAtta To ensure no further Travis builds are triggered when a map maker pushes a new commit, all map repos should be disabled at https://travis-ci.org/profile/triplea-maps.  I know of no other way to do this than to flip the toggles to "X" (disabled) on all 12 pages of the profile at the above link (and only someone with admin access can do that). 😞

**Comment by sneakingcoward** [comment]
all ok..

 

 

Gesendet: Mittwoch, 13. Dezember 2017 um 16:43 Uhr
Von: "Steven Soloff" <notifications@github.com>
An: triplea-maps/Project <Project@noreply.github.com>
Cc: sneakingcoward <wgbt@gmx.at>, Mention <mention@noreply.github.com>
Betreff: Re: [triplea-maps/Project] Travis still attempting to build some map repos (#89)


@FrostionAAA Yes, I now have push access to all repos, so there's nothing that you need to do. Sorry for the poke. 😄

@sneakingcoward The changes I will be submitting will not touch any map resources. They will only delete the .travis.yml and build.gradle files. If you'd prefer to do that yourself, please let me know which repos you manage, and I'll exclude them from my script. Otherwise, I plan on doing a mass change to all repos sometime today.

**Comment by ssoloff** [comment]
@ron-murhammer @DanVanAtta Bumping [this comment](https://github.com/triplea-maps/Project/issues/89#issuecomment-351607475).

**Comment by ssoloff** [comment]
@ron-murhammer @DanVanAtta Bumping [this comment](https://github.com/triplea-maps/Project/issues/89#issuecomment-351607475).

**Comment by ron-murhammer** [comment]
@ssoloff It appears they are all disabled. The ones that are enabled appear to point to non-existent repos.

**Comment by ssoloff** [comment]
@ron-murhammer Hmm, as you said, the `triplea-maps/new_world_order-skin_gudkarma_1` build is disabled, but Travis has tried to build it (and errored) after the last two bulk PRs I've submitted during the past two months (see the list [here](https://travis-ci.org/triplea-maps/new_world_order-skin_gudkarma_1/builds)).  I had just assumed there was an errant toggle enabled someplace triggering this map's build.  Let me try digging a little deeper...

**Comment by ssoloff** [comment]
@ron-murhammer I can't find any other reason why Travis continues to build `triplea-maps/new_world_order-skin_gudkarma_1` when it appears to be disabled.

I went to the Settings page and enabled "Build only if .travis.yml is present."  I verified that, now, when  I push a change, Travis no longer attempts to build the project.  Not sure why this is necessary, but it solves the immediate issue.

With that, I think this issue can now be closed.
